shell/plugins/steve/subscribe.js

```diff
@@ -323,6 +370,12 @@ export const actions = {
       msg.selector = selector;
     }
 
+    if (worker.mode === 'advanced') {
```

Comment:
it'll always be false and therefore the watch will always eventually make it to the worker where it'll check to see if it's watching or not there. That being said, I went ahead and made some modifications so that when we start a watch we signal back to the UI thread so the watchStarted getter will be accurate.